### ISSUE TITLE
Operation name missing in portal when calling trackDependency

### ISSUE THREAD

**Issue by willbar** [Microsoft/ApplicationInsights-node.js#58]
I'm calling trackDependency to track calls to Azure Storage and although I'm passing an operation name as the second parameter it's not appearing in the portal:

```
metricsClient.trackDependency(blobService.host.primaryHost, 'listBlobsSegmented', (new Date().getTime() - started), !err);
```

<img width="534" alt="screen shot 2015-11-30 at 1 05 18 pm" src="https://cloud.githubusercontent.com/assets/3266447/11484503/24016840-9763-11e5-9e46-7c71cd383a85.png">

**Comment by SergeyKanzhelev** [comment]
where do you see this chart? Operation name may refer to .Context.Operation.Name - the name of the page this dependency was tracked for.

**Comment by willbar** [comment]
I see it when I click Server Requests in the Overview Timeline and then click one of the dependency rows in the Total of Dependency Calls by Dependency table. Your response makes sense since my requests are also missing a name, I expected the requests auto-collector to populate the operation name, is there something I need to do to make that happen?

**Comment by SergeyKanzhelev** [comment]
Yes, it is indeed the name of the page should be there. I don't know how `request.name` being populated in node.js SDK and whether we associate dependency calls with operation name. @scsouthw , @BogdanBe do you know?

Definition of this field is clearly there: https://github.com/Microsoft/ApplicationInsights-node.js/blob/3a3078556e85be2d4117f28151270726c90b96bf/Library/Contracts.ts#L91

**Comment by willbar** [comment]
I see request.name being set in RequestDataHelper when I set a breakpoint but I don't see any server requests with an operation name in the portal (however there are server requests because I see them in the graph):

<img width="528" alt="screen shot 2015-11-30 at 3 10 06 pm" src="https://cloud.githubusercontent.com/assets/3266447/11487511/932aa2fc-9774-11e5-8f36-50ae885a5848.png">

I wouldn't expect the dependency tracking to have the operation name since I'm calling that API directly, if we can get the operation name showing for the server requests I should be able to get it to carry through to the dependencies.

**Comment by scsouthw** [comment]
@SergeyKanzhelev, should the request auto-collection be setting the ```operationName```? If so we could probably make that happen by adding it to the ```tags``` object here: https://github.com/Microsoft/ApplicationInsights-node.js/blob/f1cf21959128526ad2d4fbd357de46d55fa5a729/AutoCollection/Requests.ts#L88

**Comment by willbar** [comment]
@SergeyKanzhelev is this the right fix? If so, I'm happy to implement and test it out in my environment to see if it resolves the issue. Not having the operations listed limits the usefulness pretty significantly.

**Comment by SergeyKanzhelev** [comment]
Yes, it will be great to set OperationName for proper correlation with the request

**Comment by irakliy81** [comment]
I'm experiencing a similar issue - though, for me 'operation name' does not get set for requests as well (not just dependencies).

**Comment by kamilszostak** [comment]
I sent a pull request with a fix: #97. The fix will be included in the next SDK release.
